job-machine/src/utils/utils.ts

```diff
@@ -89,3 +89,14 @@ export const handleErrorShow = (error: any) => {
 export const handleSuccessShow = (response: AxiosResponse<axiosApi>) => {
   const { message } = handleSuccess(response);
 };
+
+export const getClassForCol = col => {
```

Comment:
hàm này không dùng cho toàn bộ app, nên có thể để trong cùng file.
func getClassForCol {
const colClasses = {
  24: 'xl-video-player',
  6: 'md-video-player',
};

return colClasses[col] || 'xs-video-player';}

Tham khảo hàm này xem cái nào clear hơn